ros2_ws/src/slu_smart/setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import find_packages, setup
+
+package_name = 'slu_smart'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='pascal',
```

Comment:
do we want to register a group e-mail through SLU until a proper domain is set up and ready to use?